src/vitepress/config/baseConfig.js

```diff
@@ -35,7 +35,7 @@ module.exports = async () => ({
             'link',
             {
               rel: 'preload',
-              href: '/assets/inter-latin.7b37fe23.woff2',
+              href: '/assets/inter-latin.4fe6132f.woff2',
```

Comment:
May I know where this hash comes from? And it would be better to leave some comments here about that I think.

Reply:
That font is imported in the fonts.css file. Vite/Rollup adds hash to assets to allow their caching. If this is not updated, there will be a failed preload request showing up in the console (dev tools). Also, IG this will need changing again if we bump VitePress version to alpha-31+. In newer versions of Vite/Rollup, they are using a different hashing algorithm.